src/coreclr/nativeaot/System.Private.CoreLib/src/System/Reflection/Runtime/General/Dispensers.cs

```diff
@@ -43,35 +43,15 @@ internal static RuntimeAssembly GetRuntimeAssembly(RuntimeAssemblyName assemblyR
         /// </summary>
         internal static RuntimeAssembly GetRuntimeAssemblyFromByteArray(ReadOnlySpan<byte> rawAssembly, ReadOnlySpan<byte> pdbSymbolStore)
```

Comment:
Delete these method and inline the throw to Assembly.Load that's the only place they are called from?

Reply:
This feels like a style thing. There's lots of static methods in Dispenser.cs that only have one callsite. The pattern seems to be: this file dispends, ReflectionAugments validate arguments before that. One could inline it all. It also doesn't hurt much to just keep it the same.

Reply:
I see this style as a vestige of the early .NET Native architecture. It is not followed consistently, and there is no reason to follow it, in particular for methods that just end up throwing anyway. For example, `ModuleHandle` throws PNSE directly without going through ReflectionAugments.